examples/schelling/model.py

```diff
@@ -16,6 +16,7 @@ def __init__(self, pos, model, agent_type):
            agent_type: Indicator for the agent's type (minority=1, majority=0)
         """
         super().__init__(pos, model)
+        self.unique_id = unique_id
```

Comment:
The first argument in L18 (`pos`) is the `self.unique_id`: https://github.com/projectmesa/mesa-examples/pull/101/files#diff-a04f58e21a9365b299c78a67c883c87286b877ef1f50fffce5534d3ec3eea566R54.

Reply:
https://github.com/projectmesa/mesa/blob/6dbe3e8572db4fa0b0db6fee1ee89d4123587ad4/mesa/agent.py#L39-L47